docs/docs/zh/guide/start/quick-start.md

```diff
@@ -50,11 +50,11 @@
   * 点击工作流定义->创建工作流定义->上线工作流定义
 
 <p align="center">
-   <img src="../../../../img/dag1.png" width="60%" />
+   <img src="../../../../img/process_definition_en.png" width="60%" />
```

Comment:
As we have discussed here before: https://github.com/apache/dolphinscheduler/issues/10257#issuecomment-1175126227, what about renaming `process_definition_en` to `workflow_definition_en`?

Reply:
do you mean rename `process_definition_en` to `process_definition` ?

Reply:
> do you mean rename `process_definition_en` to `process_definition` ?

oh, my bad, you mean `process_definition` vs `workflow` ? we should do it, but I think it is better to do it in separate PR, and change other related content, WDYT

Reply:
Sure, sounds good to me.